pkgs/applications/networking/browsers/webmacs/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, qtbase
+, herbstluftwm
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "webmacs";
+  version = "0.8";
+
+  src = fetchFromGitHub {
+    owner  = "parkouss";
+    repo   = "webmacs";
+    rev    = "d692ae8c2bf82cce854f34cb3d762932fe7fdc8b";
```

Comment:
You can also just use `rev = version` here

Reply:
Nice! but how do I test it, with `fetchFromGitHub` only changing when `sha256` changes?

Reply:
set sha256 to `lib.fakeSha256`, run the build, it'll tell you the hash it was expecting and then you just stick that in